MIGRATION.md

```diff
@@ -0,0 +1,324 @@
+
+# Brief overview of Changes in Components
+
+Splunk Connect for Kubernetes(SCK) has 3 components/applications:
+1.  Application to fetch logs from a kubernetes cluster (deployed as a DaemonSet)
+2.  Application to fetch metrics from a kubernetes cluster (deployed as a DaemonSet)
+3.  Application to fetch kubernetes objects metadata from a kubernetes cluster (deployed as a deployment)
+
+All the applications in SCK use Fluentd to work with logs, metrics and objects. There are performance limitations associated with Fluentd when using the logging application to fetch logs in a kubernetes cluster with pods that have very high throughput. To solve this issue we changed the implementation of the application to use Opentelemetry agent instead of Fluentd for high throughput scenarios. You can learn more about the performance characteristics of this new application [here](https://github.com/splunk/sck-otel#performance-of-splunk-connect-for-kubernetes-opentelemetry). If you want to learn more about the application architecture, we have shared a brief summary [here](https://github.com/splunk/sck-otel#performance-of-splunk-connect-for-kubernetes-opentelemetry).
+
+Splunk Connect for Kubernetes Opentelemetry(SCK-OTEL) has 2 components/applications:
+1.  Application to fetch logs and traces from a kubernetes cluster (deployed as a DaemonSet)
+2.  Application to fetch metrics and objects from a kubernetes cluster (deployed as a DaemonSet).
+
+No application currently exists for fetching kubernetes objects metadata from a kubernetes cluster. 
+## Brief overview of Changes in Logs in Splunk Connect for Kubernetes Opentelemetry(SCK-OTEL)
+* Redhat UBI docker images for our applications are no longer available, as we now use scratch images.
+* SCK-OTEL does not support AWS Firelens. AWS Firelens uses Fluentd and we can achieve similar outcomes with opentelemetry.
+
+## Brief overview of Changes in Metrics in Splunk Connect for Kubernetes Opentelemetry(SCK-OTEL)
+* The naming convention of the metrics used in SCK-OTEL has changed and you will observe some minor differences in the names of the metrics. 
+* Previously in SCK, you could get a large number of metrics from various APIs for kubernetes. However, in recent versions of kubernetes 1.18+, these API sources are disabled by default and there are fewer metrics available. The previous list of metrics can be found [here](https://github.com/splunk/fluent-plugin-kubernetes-metrics/blob/master/metrics-information.md).
+
+Some additional metrics that are available in SCK-OTEL are metrics about the open telemetry collector itself.
+
+These are the metrics available in SCK-OTEL:
+* ontainer.cpu.time
```

Comment:
the first letter is missed here and in other metrics

Reply:
updated